language/analyze.js

```diff
@@ -163,59 +173,62 @@ function analyzeSyntaxInFile (bucketName, fileName) {
   return document.detectSyntax()
     .then((results) => {
       const syntax = results[0];
+
       console.log('Tags:');
       syntax.forEach((part) => console.log(part.tag));
+
       return syntax;
     });
 }
 // [END language_syntax_file]
 
-// The command-line program
-const cli = require(`yargs`);
-
-const program = module.exports = {
-  analyzeSentimentOfText,
-  analyzeSentimentInFile,
-  analyzeEntitiesOfText,
-  analyzeEntitiesInFile,
-  analyzeSyntaxOfText,
-  analyzeSyntaxInFile,
-  main: (args) => {
-    // Run the command-line program
-    cli.help().strict().parse(args).argv;
-  }
-};
-
-cli
+require(`yargs`)
   .demand(1)
-  .command(`sentimentOfText <text>`, `Detect the sentiment of a block of text.`, {}, (opts) => {
-    program.analyzeSentimentOfText(opts.text);
-  })
-  .command(`sentimentInFile <bucket> <filename>`, `Detect the sentiment of text in a GCS file.`, {}, (opts) => {
-    program.analyzeSentimentInFile(opts.bucket, opts.filename);
-  })
-  .command(`entitiesOfText <text>`, `Detect the entities of a block of text.`, {}, (opts) => {
-    program.analyzeEntitiesOfText(opts.text);
-  })
-  .command(`entitiesInFile <bucket> <filename>`, `Detect the entities of text in a GCS file.`, {}, (opts) => {
-    program.analyzeEntitiesInFile(opts.bucket, opts.filename);
-  })
-  .command(`syntaxOfText <text>`, `Detect the syntax of a block of text.`, {}, (opts) => {
-    program.analyzeSyntaxOfText(opts.text);
-  })
-  .command(`syntaxInFile <bucket> <filename>`, `Detect the syntax of text in a GCS file.`, {}, (opts) => {
-    program.analyzeSyntaxInFile(opts.bucket, opts.filename);
-  })
-  .example(`node $0 sentimentOfText "President Obama is speaking at the White House."`, ``)
-  .example(`node $0 sentimentInFile my-bucket file.txt`, ``)
-  .example(`node $0 entitiesOfText "President Obama is speaking at the White House."`, ``)
-  .example(`node $0 entitiesInFile my-bucket file.txt`, ``)
-  .example(`node $0 syntaxOfText "President Obama is speaking at the White House."`, ``)
-  .example(`node $0 syntaxInFile my-bucket file.txt`, ``)
+  .command(
+    `sentiment-text <text>`,
+    `Detects sentiment of a string.`,
+    {},
+    (opts) => analyzeSentimentOfText(opts.text)
+  )
+  .command(
+    `sentiment-file <bucket> <filename>`,
+    `Detects sentiment in a file in Google Cloud Storage.`,
+    {},
+    (opts) => analyzeSentimentInFile(opts.bucket, opts.filename)
```

Comment:
I am not seeing any output from $ node analyze sentiment-text "President Obama is speaking" for example.